### DISCUSSION TITLE
Low speeds on WiFi compared to ethernet

### DISCUSSION THREAD

**Discussion by jodave911** [valnoxy/t480-oc#41] [Q&A]
![Screen Shot 2022-09-08 at 1 03 05 AM](https://user-images.githubusercontent.com/62954129/188962674-834cf4d3-9afd-47a5-9a56-93ef48648883.png)
This is my speed while im connected on eth on the same router. 
This should be the actual speed. 
But instead i'm getting low speed.
![Screen Shot 2022-09-08 at 1 05 39 AM](https://user-images.githubusercontent.com/62954129/188962872-32e97375-2077-4257-9b26-c6f19859e81f.png)
This the speed i'm getting while i'm connected on the WiFI.

- Joshua D

**Comment by Real0ver** [comment]
Is there any difference on Windows/Linux?
